### ISSUE TITLE
Why haven't we utilized single-cell proteomics for cell communication recognition and prediction?

### ISSUE THREAD

**Issue by YuzhiSun** [sqjin/CellChat#740]
Dear Suoqin，
I would like to ask you a question：Why haven't we utilized single-cell proteomics for cell communication recognition and prediction? Isn't protein a more direct reflection of the final protein expression? As the direct actors in cell communication, can't they provide a more accurate prediction of cellular communication?

Best wishes,
Yuzhi

**Comment by sqjin** [comment]
@YuzhiSun Yes, you are right. But the current technique of single-cell proteomics can only capture hundreds of proteins, among which there are only a few signaling molecules.
